### PR TITLE
added SSP playbook

### PR DESCRIPTION
**Pull request by Knickkennedy**
attempt deuce with the sign-off added

### CODE REVIEW

**Comment by Knickkennedy** [comment]
Okay let's try this again!

**Comment by hollisc** [comment]
Please resolve existing conflict and this will have to wait until the other PRs are merged.

File: 0-bootstrap/single-cluster/1-infra/argocd/namespace-sterling-sp.yaml
Comment (hollisc) [inline 0-bootstrap/single-cluster/1-infra/argocd/namespace-sterling-sp.yaml]: The path should be `namespaces/ssp`

File: 0-bootstrap/single-cluster/1-infra/kustomization.yaml
Comment (hollisc) [inline 0-bootstrap/single-cluster/1-infra/kustomization.yaml]: Adjust the name to match the namespace so it should be `namesapce-ssp.yaml`

**Review by Knickkennedy — COMMENTED** [review]
I think this fixes everything @hollisc

**Review by hollisc — APPROVED** [review]
Looks good

**Pull request merged**
